docs/index.rst

```diff
@@ -61,6 +61,75 @@ Quick Start
 
    .. note:: It is always recommended to use python virtual environment
 
+.. _how_it_works:
+
+How it works?
+`````````````
+Assuming that you have a ``test_user.py`` file with the following content:
+
+.. code-block:: python
+
+    import entities
+    import unittest
+
+
+    class EntitiesTest(unittest.TestCase):
+
+        def test_positive_create_user(self):
+            user = entities.User(name='David', age=20)
+            self.assertEqual(user.name, 'David')
+            self.assertEqual(user.age, 20)
+
+        def test_positive_create_car(self):
+            car = entities.Car(make='Honda', year=2016)
+            self.assertEqual(car.make, 'Honda')
+            self.assertEqual(car.year, 2016)
+
+Using the example above, Betelgeuse will recognize that there are 2 test cases
+available, and the following attributes will be derived:
+
+* Name: this attribute will be derived from the name of the test method itself
+
+      - test_positive_create_user
+      - test_positive_create_car
+
+* ID: this attribute will be derived from the concatenation of the
+  *module.test_name* or *module.ClassName.test_name* if the test method is
```

Comment:
I think that could be added _in other words the Python import path will be used_.